db/migrate/20220513185014_add_user_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] According to the ERD diagram, the `comment` table should have `AuthorId` instead of `user_id`. Kindly add `AuthorId` and refer it to the user table like this.
- `t.references :author, foreign_key: { to_table: 'users' }    # this will create author_id and a corresponding index`